fdbserver/ClusterController.actor.cpp

```diff
@@ -337,9 +337,84 @@ class ClusterControllerData {
 				else {
 					std::vector<LocalityEntry> bestSet;
 					std::vector<LocalityData> tLocalities;
+					bool bestFound = false;
 
 					// Try to find the best team of servers to fulfill the policy
-					if (findBestPolicySet(bestSet, logServerSet, policy, desired, SERVER_KNOBS->POLICY_RATING_TESTS, SERVER_KNOBS->POLICY_GENERATIONS)) {
+
+					// Specialization for policies of shape:
+					//    - PolicyOne()
+					//    - PolicyAcross(,"zoneId",PolicyOne())
+					//    - TODO: More specializations for common policies
+					if (policy->name() == "One") {
```

Comment:
Could you just move these two special cases into findBestPolicySet so that other users of it also benefit?  Is this a commentary that findBestPolicySet is too slow?

Reply:
The only usage of `findBestPolicySet` is here in cluster controller. That being said, it seems reasonable to move these code into the function itself. @etschannen What do you think?

>  Is this a commentary that findBestPolicySet is too slow

We have some Net2SlowTaskTrace events which pointing to this function.

Reply:
I agree that we should move it